Dockerfile

```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
 COPY config /root/.kube/config
 COPY settings.xml /root/.m2/settings.xml
 
-CMD bash
\ No newline at end of file
+CMD bash
+
+FROM node:alpine
```

Comment:
docker must be light, please do not extend from any others, just add required dependency

Reply:
+